app/js/services/resultatService.js

```diff
@@ -9,7 +9,8 @@ angular.module('ddsApp').service('ResultatService', function($http, droitsDescri
     *@return   {Object}  A new object containing the ressources of the family and of the individual. The family ressources will be overridden if conflicting.
     */
     function normalizeOpenfiscaRessources(testCase) {
-        return _.merge({}, testCase.menages._, testCase.familles._, testCase.individus.demandeur);
+        var individuId = testCase.menages._.personne_de_reference[0];
```

Comment:
Is there always at least 1 element in the array?

Reply:
Well, yes, in all valid situation. Worst case scenario, arror should be capture in the catch section.